### ISSUE TITLE
5-2-stable CI GEM=ar:postgresql with Ruby2.5.1 has been failing

### ISSUE THREAD

**Issue by yahonda** [rails/rails#34664]
### Steps to reproduce
5-2-stable CI GEM=ar:postgresql with Ruby2.5.1 has been failing since https://travis-ci.org/rails/rails/jobs/463484520

The last successful one is https://travis-ci.org/rails/rails/jobs/463181829

### Expected behavior
5-2-stable CI GEM=ar:postgresql with Ruby2.5.1 should pass.

### Actual behavior
```ruby
/home/travis/.rvm/rubies/ruby-2.5.1/bin/ruby -w -Itest test/cases/relation/merging_test.rb
/home/travis/build/rails/rails/activerecord/lib/active_record/connection_adapters/postgresql/database_statements.rb:75:in `async_exec': PG::UnableToSend: server closed the connection unexpectedly (ActiveRecord::StatementInvalid)
	This probably means the server terminated abnormally
	before or while processing the request.
: CREATE TABLE "essays" ("id" bigserial primary key, "name" character varying, "writer_id" character varying, "writer_type" character varying, "category_id" character varying, "author_id" character varying)
	from /home/travis/build/rails/rails/activerecord/lib/active_record/connection_adapters/postgresql/database_statements.rb:75:in `block (2 levels) in execute'
	from /home/travis/build/rails/rails/activesupport/lib/active_support/dependencies/interlock.rb:48:in `block in permit_concurrent_loads'
	from /home/travis/build/rails/rails/activesupport/lib/active_support/concurrency/share_lock.rb:187:in `yield_shares'
	from /home/travis/build/rails/rails/activesupport/lib/active_support/dependencies/interlock.rb:47:in `permit_concurrent_loads'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/connection_adapters/postgresql/database_statements.rb:74:in `block in execute'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb:579:in `block (2 levels) in log'
	from /home/travis/.rvm/rubies/ruby-2.5.1/lib/ruby/2.5.0/monitor.rb:226:in `mon_synchronize'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb:578:in `block in log'
	from /home/travis/build/rails/rails/activesupport/lib/active_support/notifications/instrumenter.rb:23:in `instrument'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/connection_adapters/abstract_adapter.rb:569:in `log'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/connection_adapters/postgresql/database_statements.rb:73:in `execute'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/connection_adapters/abstract/schema_statements.rb:311:in `create_table'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/migration.rb:871:in `block in method_missing'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/migration.rb:840:in `block in say_with_time'
	from /home/travis/.rvm/rubies/ruby-2.5.1/lib/ruby/2.5.0/benchmark.rb:293:in `measure'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/migration.rb:840:in `say_with_time'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/migration.rb:860:in `method_missing'
	from /home/travis/build/rails/rails/activerecord/test/schema/schema.rb:327:in `block in <top (required)>'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/schema.rb:50:in `instance_eval'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/schema.rb:50:in `define'
	from /home/travis/build/rails/rails/activerecord/lib/active_record/schema.rb:46:in `define'
	from /home/travis/build/rails/rails/activerecord/test/schema/schema.rb:3:in `<top (required)>'
	from /home/travis/build/rails/rails/activesupport/lib/active_support/dependencies.rb:285:in `load'
	from /home/travis/build/rails/rails/activesupport/lib/active_support/dependencies.rb:285:in `block in load'
	from /home/travis/build/rails/rails/activesupport/lib/active_support/dependencies.rb:257:in `load_dependency'
	from /home/travis/build/rails/rails/activesupport/lib/active_support/dependencies.rb:285:in `load'
	from /home/travis/build/rails/rails/activerecord/test/cases/helper.rb:141:in `load_schema'
	from /home/travis/build/rails/rails/activerecord/test/cases/helper.rb:152:in `<top (required)>'
	from test/cases/relation/merging_test.rb:3:in `require'
	from test/cases/relation/merging_test.rb:3:in `<main>'
Using postgresql
rake aborted!
```
### System configuration
**Rails version**:5-2-stable

**Ruby version**:2.5.1

**Comment by yahonda** [comment]
Related to these issues/commits:

https://github.com/rails/rails/issues/31369
https://github.com/rails/rails/pull/33858#event-1842726050
https://github.com/rails/rails/commit/82556cd452563abcf985ca33d1c82dfa3d4fc443#diff-354f30a63fb0907d4ad57269548329e3

@y-yagi What do you think is the better way to address 5-2-stable CI failures?

**Comment by yahonda** [comment]
I prefer https://github.com/rails/rails/commit/1b5bae4050be26658fdd3aa06fd353d30a58408c for 5-2-stable but wanted to hear from another opinion.

**Comment by y-yagi** [comment]
I think it is good to increase ramfs. 
But before merge, I would like to check if the isolation test passes if I do it. Can you create a PR for verification?

**Comment by yahonda** [comment]
Addressed by #34669
